app/app.go

```diff
@@ -138,7 +138,7 @@ func HealthHandler(w http.ResponseWriter, r *http.Request, jobsDB jobsdb.JobsDB)
 	var dbService string = "UP"
 	var enabledRouter string = "TRUE"
 	var backendConfigMode string = "API"
-	if !jobsDB.CheckPGHealth() {
+	if !jobsDB.IsHealthy() {
```

Comment:
Since we change the name of the method, I will suggest naming it `.Ping` and return an error instead of bool

```go
Ping() error
```

Reply:
It is not just a ping we perform here... but I like the ping error approach.